pkg/apis/options/legacy_options.go

```diff
@@ -510,10 +510,12 @@ type LegacyProvider struct {
 	AllowedGroups                      []string `flag:"allowed-group" cfg:"allowed_groups"`
 	AllowedRoles                       []string `flag:"allowed-role" cfg:"allowed_roles"`
 
-	AcrValues  string `flag:"acr-values" cfg:"acr_values"`
-	JWTKey     string `flag:"jwt-key" cfg:"jwt_key"`
-	JWTKeyFile string `flag:"jwt-key-file" cfg:"jwt_key_file"`
-	PubJWKURL  string `flag:"pubjwk-url" cfg:"pubjwk_url"`
+	AcrValues       string `flag:"acr-values" cfg:"acr_values"`
+	Certificate     string `flag:"certificate" cfg:"certificate"`
+	CertificateFile string `flag:"certificate-file" cfg:"certificate-file"`
```

Comment:
Would recommend that we rename these to include a `client` prefix to make sure it's obvious that this is part of the OAuth2/OIDC provider code